udata/tests/api/test_dataservices_api.py

```diff
@@ -192,6 +244,9 @@ def test_dataservice_api_index(self):
 
         self.assertEqual(Dataservice.objects.count(), 4)
 
+        # Login with a distinct user
+        self.login(UserFactory())
+
```

Comment:
What is that for? I'm also not sure what the assertion below checks, other than "this url exists"?

Reply:
If we don't change the User we're logged in with, we would have dataservice being returned since the user would now get the private dataservice in the response.
I've updated the comment